fossunited/fossunited/user_utils.py

```
@@ -13,31 +15,119 @@ def set_unique_username(doc, method):
-            }
+    if doc.user_type == "Website User":
+        # Disable until approved
+        frappe.db.set_value("User", doc.name, "enabled", 0, update_modified=False)
+
+        # Notify user: pending
+        frappe.sendmail(
+            recipients=[doc.email],
+            subject="Your FOSS United account is pending approval",
+            message=f"""<p>Hi {frappe.utils.escape_html(doc.full_name)},</p>
+<p>Thank you for signing up on FOSS United! Your account is currently <strong>pending review</strong> by our team.</p>
+<p>We'll email you once approved. This usually takes 1–2 working days.</p>
```

Ruff (0.15.10)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)